snapshots/devmapper/snapshotter_test.go

```diff
@@ -137,6 +138,9 @@ func TestMkfsXfsNonDefault(t *testing.T) {
 
 func TestMultipleXfsMounts(t *testing.T) {
 	testutil.RequiresRoot(t)
+	if goruntime.GOARCH == "arm64" {
```

Comment:
It would be better to pass something through from the workload.  This test should still run on non-Actuated arm64 hosts.

Reply:
Removed the skipf

Reply:
Xfs is available now per @fuweid's request..

Howevr, to @samuelkarp's point - perhaps it'd be better to parse the Kernel config to work out what's available or detect it via /sys/ or /proc etc?

Reply:
Hi @alexellis I miss this comment. Basically, containerd mount related test cases can skip if it doesn't support XFS. However, for device mapper snapshot component, I want to run that test case for XFS. XFS is general FS for user. It's worth to keep it.